samples/README.md

```diff
@@ -6,28 +6,21 @@
 Use the extension proposed in the readme.
 
 ### 3. Database
-The docker-compose deploys a database, but it is empty. To have some data to extract, you need to fill it a bit:
+The docker-compose deploys a database, and deploys some sample data : it creates foncier_2014 schema and pushes some cadaster data on it
 
- - create schemas foncier_2014, foncier_2015 etc (one at least). The years must match with the header config.
-
- - push some data inside. The app will make exports of pretty much every table in the database, but to get some data, the tables must have a 'idcom' attribute, which values will have to match (at least for some entries) with the zones listed in the LDAP entry.
- You can push the /samples/parcelles.sql file, containing some cadaster data on some communes in Ariege. The idcom attribute has been properly adusted. It will push the shapefile _in the foncier_2014 schema_
- ```psql -h localhost -p 54322 -U geoserver -d geoserver -f parcelles.sql```
-(password _geoserver_)
- - adjust the LDAP settings: log into the LDAP container (```docker ps |grep ldap  then docker xec -it LDAP_CONTAINER_ID /bin/bash```) and then, change the active zone for PSC organization (description lists the idcom values accepted for this profile):
+To be eligible to download this data, you need to adjust the LDAP settings: log into the LDAP container (```docker ps |grep ldap  then docker xec -it LDAP_CONTAINER_ID /bin/bash```) and then, change the active zone for PSC organization (description lists the idcom values accepted for this profile):
```

Comment:
still required ?

Reply:
not anymore since you approved my PR on georchestra ldap

Reply:
OK, done. But I've added a paragraph in the main README file. I feel it might help people test and use the app.